packages/office-ui-fabric-react/src/components/OverflowSet/OverflowSet.base.tsx

```diff
@@ -142,11 +145,8 @@ export class OverflowSetBase extends BaseComponent<IOverflowSetProps, {}> implem
 
   private _onRenderItems = (items: IOverflowSetItemProps[]): JSX.Element[] => {
     return items.map((item, i) => {
-      const wrapperDivProps: React.HTMLProps<HTMLDivElement> = {
-        className: this._classNames.item
-      };
       return (
-        <div key={item.key} {...wrapperDivProps}>
+        <div role="presentation" key={item.key} className={this._classNames.item}>
```

Comment:
>presentation [](start = 19, length = 12)

Note, you don't have to put role=presentation here. role=presentation just makes the HTML element not have the default role (<li> is an example, it has role=listitem) and div doesn't have one unless it is made an accessible node

Reply:
ok...so it's only to change something that'd have a default value applied/assumed for it.

Reply:
wait...but, doesn't the menubar expect its children to be menuitems?

Reply:
Yes, the focusable elements should be a menuitem-type role

Reply:
the focusable element will be the control rendered inside of this wrapper div (button, link etc). So if that button has role="menuitem", then this div should have role="presentation", since it is the direct child of the root, when it has role="menu".

I could make the above role conditional to role="menubar" on the root, but is there harm in having it here?

Reply:
see example 8 in the aria spec for [role=presentation](https://www.w3.org/TR/wai-aria-1.1/#presentation), `<span>` is an example of what I'm talking about (e.g. `<div>` does not have an explicit role (unless it is given an accessible name) and will be ignored). Historically, there's been an overuse of role presentation when it's not needed.

For example, I put together a [jsfiddle](https://jsfiddle.net/6a0ubhw1/) showing two button, one wrapped in a div. Note the resulting accessibility tree:
![image](https://user-images.githubusercontent.com/7033561/72389606-9e50f700-36dd-11ea-8a57-4ecb3e1d81c1.png)

Reply:
Thanks for the clarification. I agree, I have seen 'presenation' used a ton (details list for example), so I expected it was needed here. I can remove it.